packages/mui/src/pickers-common/MultiLocalizationProvider.tsx

```diff
@@ -10,20 +10,53 @@ import {
   defaultPersianLocaleTexts,
 } from './default-locale-texts';
 
+type GeneralizedLocale = Locale | string;
```

Comment:
why `| string`

Reply:
GeneralizedLocale is properly implemented.